CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ cmake_minimum_required(VERSION 2.6)
 add_definitions(-D UNIX)
 add_definitions(-D LINUX)
 add_definitions(-D _GNU_SOURCE)
+add_definitions(-std=c++11 -fpermissive)
```

Comment:
I've updated my pull request #53 to make the ssl-vision codebase compatible with both C++98 and C++11.